tests/playbooks/deb_sync.yaml

```diff
@@ -91,6 +121,10 @@
   module_defaults:
     <<: *pulp_module_defaults
   tasks:
+    - name: Make repository with default remote absent
```

Comment:
We can use a looping pattern here as well as in the at the beginning of the setup.
Also I would remove the default from the name. It's a repository with a remote.
```yaml
    - name: "Make repositories absent"
      pulp.squeezer.deb_repository:
        name: "{{ item }}"
        state: "absent"
      loop:
        - "test_deb_repository"
        - "test_deb_repository_with_remote"
```